controllers/rootpolicystatus/filters.go

```diff
@@ -28,3 +36,56 @@ func policyStatusPredicate() predicate.Funcs {
 		DeleteFunc: func(e event.DeleteEvent) bool { return false },
 	}
 }
+
+// mapBindingToPolicies maps a placementBinding to all the Policies in its policies list.
+func mapBindingToPolicies(c client.Client) handler.MapFunc {
+	return func(ctx context.Context, object client.Object) []reconcile.Request {
+		log := log.WithValues("placementBinding", object.GetName(), "namespace", object.GetNamespace())
+		log.V(2).Info("Reconcile Request for placementBinding")
+
+		//nolint:forcetypeassert
+		pb := object.(*policiesv1.PlacementBinding)
+
+		return common.GetPoliciesInPlacementBinding(ctx, c, pb)
```

Comment:
It occurred to me that that this function doesn't ignore the placement bindings that have `SubFilter` set to `Restricted`. @JustinKuli , what are you thoughts on this?

Reply:
It shouldn't just ignore them always though, right? I think it would need to lookup other bindings for each candidate policy to try and determine what is correct... which is the calculation the root policy reconcile would have to make anyway. So I'm in favor of ignoring the subfilter here, and possibly flagging some policies to be reconciled that don't strictly need to be.

I know the whole intent with the epic is to reduce unnecessary reconciles, but I don't think we need to guarantee that we now do the absolutely minimum number of reconciles. Especially when incorrectly skipping something (because of a bug or oversight) will cause other problems.

Reply:
Yeah, this should be light-weight enough since it's just the root policy status reconciler and if nothing changed, there's no HTTP request made.